gears/src/application/handlers.rs

```diff
@@ -54,24 +55,30 @@ pub trait TxHandler {
             account_number: account.account.get_account_number(),
         };
 
-        let tx_body = TxBody {
-            messages: vec![msg],
-            memo: String::new(),                    // TODO: remove hard coded
-            timeout_height: 0,                      // TODO: remove hard coded
-            extension_options: vec![],              // TODO: remove hard coded
-            non_critical_extension_options: vec![], // TODO: remove hard coded
-        };
-
+        let fee = Fee::new_with_defaults(fee);
```

Comment:
I'm not sure how valuable this method will be once we add more params to the CLI?

Reply:
It's a way to initialize instance and then setup it's fields. 

Sure, we can get the values from the CLI.

Please, add comment: Should I remove default constructors from `Fee` and `TxBody`? I'll implement changes in the next commit.